packages/transformers/js/src/visitors/dependencies.js

```diff
@@ -240,15 +240,15 @@ function addDependency(
 ) {
   asset.addDependency({
     moduleSpecifier: node.value,
-    loc: node.loc && node.loc.start,
+    loc: node.loc && createDependencyLocation(node.loc.start, node.value, 1, 1),
```

Comment:
I think these are off by one line. I'm getting this:

<img width="822" alt="image" src="https://user-images.githubusercontent.com/19409/69980679-b8a08e00-14fe-11ea-9300-6ce972ea1a50.png">

Changing `lineOffset` to `0` makes it work...

Reply:
strange it worked on my test, I'll change it and maybe add a couple tests later today or tomorrow